tests/test-pipenv.bsh

```diff
@@ -18,3 +19,39 @@ begin_test "Pipenv"
 
 )
 end_test
+
+# test 30_get-pipenv script without a container
+# Note this test assumes a system-wide python is available
+begin_test "30_get-pipenv"
+(
+  setup_test
+
+  # environment
+  export PIPENV_VERSION="2020.8.13"
+  export LC_ALL=C.UTF-8
+  export LANG=C.UTF-8
```

Comment:
Unfortunately, I don't think this is this simple...

I believe debian and fedora call UTF 8 two different ways. Debian has `C.UTF-8` by default, and fedora has: `en_US.UTF-8`

I wouldn't want to code "debian" vs "fedora" to fix this. Is there a universal way to test "is this a valid locale" in bash?

https://stackoverflow.com/a/18930395/4166604

So maybe if `command -v locale` is true, use it to determine which locale is valid. If it doesn't exist, default to `C.UTF-8`

Reply:
Uddated.